src/librustc/hir/lowering.rs

```diff
@@ -5755,7 +5757,7 @@ impl<'a> LoweringContext<'a> {
         }
     }
 
-    /// Invoked to create the lifetime argument for a type `&T`
+    /// Invoked to create the lifetime parameter for a type `&T`
```

Comment:
This was correct, it was referring to a "generic argument" (and more specifically, a lifetime).